packages/web/config/chain-infos.ts

```diff
@@ -2896,6 +2896,34 @@ const chainInfos = (
       features: ["ibc-transfer", "ibc-go"],
       explorerUrlToTx: "https://explorer.8ball.info/8ball/tx/{txHash}",
     },
+    {
+      rpc: "https://arkh-rpc.kynraze.com",
+      rest: "https://arkh-api.kynraze.com",
+      chainId: "arkh",
+      chainName: "Arkhadian",
+      bip44: {
+        coinType: 118,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("arkh"),
+      currencies: [
+        {
+          coinDenom: "ARKH",
+          coinMinimalDenom: "arkh",
+          coinDecimals: 6,
+          coinImageUrl: "/tokens/arkh.png",
```

Comment:
Let's use the svg vs the PNG, and remove the PNG

Reply:
The svg was generated by a tool that left a lot of gaps. It look quite strange, but I can re-add it if you want to see how it looks...

Reply:
PNG: 
<img width="104" alt="image" src="https://user-images.githubusercontent.com/95667791/227276715-2e3f07ba-0814-4eb1-aab0-6e4c506d867e.png">

Reply:
SVG:
<img width="97" alt="image" src="https://user-images.githubusercontent.com/95667791/227279042-61dac3ba-8135-424a-bb13-072ab9fca9bb.png">